### PR TITLE
fix imagenet example with synthetic input

### PR DESCRIPTION
**Pull request by apeforest**
Fix the example of imagenet when users runs with synthetic data. Before this fix, it would fail with error 'batch_fn' not defined.

### CODE REVIEW

**Comment by apeforest** [comment]
@yuxihu @eric-haibin-lin please help review. thanks

File: examples/mxnet_imagenet_resnet50.py [hunk 1]
```diff
@@ -136,6 +136,9 @@
 else:
     raise ValueError('Invalid lr mode')
 
+
```
Comment (alsrgv) [inline examples/mxnet_imagenet_resnet50.py]: Less whitespace?

Comment (yuxihu) [reply examples/mxnet_imagenet_resnet50.py]: +1 for less empty lines

Comment (apeforest) [reply examples/mxnet_imagenet_resnet50.py]: removed. only 2 blank lines now to satisfy pep8

**Pull request merged**